tests/testthat/helpers-with-settings.R

```diff
@@ -24,10 +24,22 @@ app_tws <- function() {
           rtables::analyze(c("SEX", "AGE"))
 
         tbl <- rtables::build_table(l, df1)
-
         tbl
       })
       table_with_settings_srv(id = "table_with_settings", table_r = table_r)
     }
   )
 }
+
+#' @keywords internal
+is_draw <- function(plot_fun) {
```

Comment:
please add some info about what this function does.

Reply:
this is a test helper function but I will add sth.